tests/test_wyoming.py

```diff
@@ -140,3 +141,14 @@ def test_wyoming_heights():
 
     assert_almost_equal(df['height'][140], 10336.0, 2)
     assert_almost_equal(df['direction'][1], 145.0, 1)
+
+
+# GH #749
+@recorder.use_cassette('wyoming_missing_station_info')
+def test_missing_station():
+    """Test that we can still return data for stations missing from the Wyoming archive."""
+    df = WyomingUpperAir.request_data(datetime(2012, 1, 1, 0),  '82244')
+    assert df['station'][0] == ''
+    assert pd.isna(df['latitude'][0])
```

Comment:
Would it be worth using `all()` to make sure all values for these variables satisfy `pd.isna`?

Reply:
Ah, good idea.

Reply:
Updated.